src/panels/lovelace/cards/hui-media-control-card.ts

```diff
@@ -460,7 +464,10 @@ export class HuiMediaControlCard extends LitElement implements LovelaceCard {
   }
 
   private _measureCard() {
-    const card = this.shadowRoot!.querySelector("ha-card")!;
+    const card = this.shadowRoot!.querySelector("ha-card");
+    if (!card) {
+      return;
```

Comment:
This is weird. If this happens, we should probably measure again when we have a card ? So `this.updateComplete.then(() => this._measureCard())`

Reply:
Otherwise we can end up never measuring.

Reply:
But if it has a card, the element will be resized so `resizeObserver` will catch that case?

Reply:
👍